Userland/Libraries/LibC/stdlib.h

```diff
@@ -49,6 +49,7 @@ char* ptsname(int fd);
 int ptsname_r(int fd, char* buffer, size_t);
 int abs(int);
 long labs(long);
+long long int llabs(long long int i);
```

Comment:
```suggestion
long long int llabs(long long int);
```

No need for the parameter name here